app/assets/stylesheets/components/_icons.css.sass

```diff
@@ -2,3 +2,11 @@
   font-family: FontAwesome
   font-size: 30px
   content: "\f0c9"
+
+.IconClose
+  display: inline-block
+
+  &:before
+    @include alpha
+    font-family: FontAwesome
```

Comment:
You should probably refactor this into a mixin. Something that accepts the content and places the font-family also.